server/routes/RoleRouter.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Dependencies declared
+ */
+import express from 'express';
+import RoleController from '../controllers/RoleController';
+import Authentication from '../middlewares/Authentication';
+/**
+ * Router for roles
+ * @type {Object}
+ */
+const roleRouter = express.Router();
+/**
+ *Routes to create and retrieve roles
+ */
+roleRouter.route('/')
+  .post(Authentication.verifyToken, Authentication.checkAdminRights,
```

Comment:
you should use .all(Authentication.verifyToken, Authentication.checkAdminRights) to avoid repetition here